static-instrumenter/agent-instrumenter/src/integrationTest/java/JarTest.java

```diff
@@ -43,17 +46,25 @@ void testSampleJar() throws Exception {
             outPath.toString());
 
     Process instrumentationProcess = instrumentationProcessBuilder.start();
-    InputStream instrumentationIn = instrumentationProcess.getErrorStream();
     instrumentationProcess.waitFor(10, TimeUnit.SECONDS);
 
-    String instrumentationLog =
-        new String(instrumentationIn.readAllBytes(), StandardCharsets.UTF_8);
+    boolean isHotJdkJvm = System.getProperty("java.vm.name").contains("OpenJDK");
+    if (isHotJdkJvm) {
+      InputStream errorOutput = instrumentationProcess.getErrorStream();
+      String errorOutputAsString = new String(errorOutput.readAllBytes(), StandardCharsets.UTF_8);
+      assertThat(errorOutputAsString)
+          .isNotEmpty()
+          .contains(
+              "Sharing is only supported for boot loader classes because bootstrap classpath has been appended");
```

Comment:
Do we actually need to assert on this string? It does not seem to be important to this scenario.

Reply:
See my comment above.

Reply:
See my comment below.